### PR TITLE
Feature/fe/poll modal fe

### PR DESCRIPTION
**Pull request by dotslash227**
1. Make sure you do yarn install.
2. To test the poll modal, you need to have valid AWS credentials, in case you want to use mine, please ask. API will fail in case of invalid AWS credentials.
3. Poll Modal is not the most beautiful modal, will be working on the prettification of it, along other tickets. :)

### CODE REVIEW

File: client/src/components/polls/PollModal.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    switch (type) {
+      case 'title':
+        setTitle(e.target.value);
+        break;
+      case 'image1':
+        console.log(e.target.files[0]);
+        setImage1(e.target.files[0]);
+        break;
+      case 'image2':
+        console.log(e.target.files[0]);
```
Comment (skassam21) [inline client/src/components/polls/PollModal.js]: Let's remove the console.logs before we merge

File: client/src/components/polls/PollModal.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    formData.set('userId', userId);
+    formData.append('image1', image1);
+    formData.append('image2', image2);
+    const config = {
+      headers: {
+        'content-type': 'multipart/form-data',
+      },
+    };
+    await axios
+      .post('http://localhost:3001/polls', formData, config)
```
Comment (skassam21) [inline client/src/components/polls/PollModal.js:52]: You don't need to specify `http://localhost:3001`

File: client/src/components/polls/PollModal.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+            style={{ marginBottom: 20 }}
+          />
+          <label>Select first image</label>
+          <TextField
+            onChange={(e) => handleChange(e, 'image1')}
+            fullWidth
+            id="standard-basic"
+            label="Upload Image 1"
+            type="file"
+            style={{ marginBottom: 20 }}
```
Comment (skassam21) [inline client/src/components/polls/PollModal.js:87]: There is any a parameter called margin for TextField to add a standard spacing between the TextFields

File: client/src/components/polls/PollModal.js (hunk 1 above)
Comment (dotslash227) [reply client/src/components/polls/PollModal.js]: Cool cool. I always forget about the console.logs :/

File: client/src/components/polls/PollModal.js (hunk 2 above)
Comment (dotslash227) [reply client/src/components/polls/PollModal.js:52]: Ok. Got it.

**Review by Hakeemmidan — APPROVED** [review]

**Pull request merged**